graphql/schemabuilder/pagination.go

```diff
@@ -380,6 +444,14 @@ func (c *connectionContext) getConnection(out []reflect.Value, args PaginationAr
 		return Connection{}, nil
 	}
 
+	if !c.IsExternallyManaged() {
+		var err error
```

Comment:
why not `nodes, err := ` below?

Reply:
We're overwriting the higher scoped nodes. I'm pretty sure that's how it works.